src/model_io/codecs/include/private/JointElement.h

```diff
@@ -59,7 +59,13 @@ class iDynTree::JointElement : public iDynTree::XMLElement {
         double velocity;
     };
 
+    struct DyamicParams {
```

Comment:
```suggestion
    struct JointDynamicParams {
```

Two changes:
* Fix typo 
* Be more precise on which type of "Dynamics" is, as iDynTree is a "dynamics" library "dynamics" is quite an overloaded term.

Reply:
Fixed the name!